.env.example

```
@@ -0,0 +1,39 @@
+# GitHub Integration
+GITHUB_TOKEN=your_github_personal_access_token
+GITHUB_REPOSITORY=techfluent-au/awesome-ai-apps
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GITHUB_REPOSITORY key should go before the GITHUB_TOKEN key

(UnorderedKey)